### DISCUSSION TITLE
Application Monitoring

### DISCUSSION THREAD

**Discussion by tobru** [appuio/appuio-cloud-community#21] [Ideas]
Allow the user of APPUiO Cloud to self-monitor the own workload running on the platform with Prometheus and Alertmanager.

[Enabling monitoring for user-defined projects](https://docs.openshift.com/container-platform/4.9/monitoring/enabling-monitoring-for-user-defined-projects.html)

**Comment by jkleinlercher** [comment]
Is there already some progress in this area? We would love to deploy workload on appuio but need to have more insights via a observability tool which collects logs, metrics and traces.

**Reply by tobru** [reply]
You find our roadmap here: https://roadmap.appuio.cloud. User workload monitoring will be available very soon :+1: You can vote on the feature on our roadmap, and you get notified once it's ready.
